src/riscv-integration.adoc

```diff
@@ -736,19 +736,22 @@ Load/store/AMO address breakpoint
 Optionally: +
 Load/store/AMO address misaligned
 | .>|13, 15, 5, 7 .<|During address translation for an explicit memory access: +
-First encountered *CHERI PTE page fault*^12^, page fault or access fault
+First encountered *CHERI PTE page fault*^23^, page fault or access fault
 | .>|5,7 .<|With physical address for an explicit memory access: +
 Load/store/AMO access fault
 | .>|4,6 .<|If not higher priority: +
 .>|_Lowest_ .>|13 .<|If not higher priority: +
-CHERI load PTE fault^3^
+CHERI load PTE fault^4^
 |===
 
-^1^ The higher priority CHERI <<cheri_pte_ext,PTE>> page fault covers capability loads or atomics where the loaded tag _is not_ checked, and all capability stores and atomics where the stored tag is set.
+^1^ PCC bounds are intended to be checked against all the bytes of fetched instructions.
+ In the case of variable length instruction encoding, and that the fetch has failed to return any data, then only a minimum length instruction is checked against the PCC bounds.
```

Comment:
This sounds a lot like “fetch the whole instruction and only then check bounds” which can be dangerous wrt side channels